packages/web-components/fast-components/src/design-tokens.ts

```diff
@@ -16,16 +17,30 @@ import {
 } from "./color/recipes/focus-stroke";
 import { neutralForeground as neutralForegroundAlgorithm } from "./color/recipes/neutral-foreground";
 import { neutralForegroundHint as neutralForegroundHintAlgorithm } from "./color/recipes/neutral-foreground-hint";
-import { neutralLayerFloating as neutralLayerFloatingAlgorithm } from "./color/recipes/neutral-layer-floating";
 import { neutralLayerCardContainer as neutralLayerCardContainerAlgorithm } from "./color/recipes/neutral-layer-card-container";
+import { neutralLayerFloating as neutralLayerFloatingAlgorithm } from "./color/recipes/neutral-layer-floating";
 import { neutralLayer1 as neutralLayer1Algorithm } from "./color/recipes/neutral-layer-1";
 import { neutralLayer2 as neutralLayer2Algorithm } from "./color/recipes/neutral-layer-2";
 import { neutralLayer3 as neutralLayer3Algorithm } from "./color/recipes/neutral-layer-3";
 import { neutralLayer4 as neutralLayer4Algorithm } from "./color/recipes/neutral-layer-4";
 import { neutralStroke as neutralStrokeAlgorithm } from "./color/recipes/neutral-stroke";
-import { SwatchRGB } from "./color/swatch";
 import { StandardLuminance } from "./color/utilities/base-layer-luminance";
 import { accentBase, middleGrey } from "./color/utilities/color-constants";
+import { SwatchFamily } from "./color/recipe";
+
+/** @public */
+// eslint-disable-next-line @typescript-eslint/ban-types
+export interface Recipe<
+    T extends string | number | boolean | BigInteger | null | Array<any> | symbol | {}
```

Comment:
This typing seems a bit odd. What is the reason for enumerating all of these types and asserting T must extend them? Can this just be `T`?

Reply:
I copied that from the `DesignToken` interface. I thought it was more than was probably necessary, but my attempt was to say that a `Recipe` could evaluate to anything that a `DesignToken` could be. Incredibly open to suggestions or a better way to do this.

Reply:
I think this can just be `T`. The limitation exist in DesignToken solely to exclude function value generics (so that it can interpret derived values), but that limitation doesn't exist here.